tutorials/marble/getting-started.md

```diff
@@ -0,0 +1,56 @@
+-[The Marble Platform](#marble-platform)
+-[Nodes](#nodes)
+- [Main Areas of the Platform](#main-areas)
+  - [The Homepage](#homepage)
+  - [JupyterLab](#jupyterlab)
+  - [The Data Catalog](#data-catalog)
+  - [The Web Processing Services](#webservices)
+
+
+## <a id="marble-platform"></a>The Marble Platform
+
+Marble provides the perfect podium for climate researchers to store and share their work. 
+A climate researcher's findings can be stored on the Marble Platform where it will be accessible by the public.
```

Comment:
It will not be available to the public. It might be if the node admin wants to set it up that way but that is not a requirement or the default configuration.

Reply:
Removed